src/FactSystem/Fact.cc

```diff
@@ -62,8 +62,6 @@ Fact::Fact(FactMetaData* metaData, QObject* parent)
     , _deferredValueChangeSignal(false)
     , _valueSliderModel         (NULL)
 {
-    // Allow core plugin a chance to override the default value
-    qgcApp()->toolbox()->corePlugin()->adjustSettingMetaData(*metaData);
```

Comment:
@dogmaphobic I also noticed this which is somewhat incorrect. In that the adjustSettingsMetaData was for settings not all Facts. At this point there is no settings group qualifier and now way to deal with name collisions. As far as I know nobody uses this ability. Only the settings overrides are used in current custom build customers.

Reply:
I'm now getting to this. Has `adjustSettingMetaData()` been removed altogether? This is used to both hide and set the (default) values of facts in custom builds. For example, if I don't want virtual joysticks, I would:

```
if (metaData.name() == AppSettings::virtualJoystickName) {
    metaData.setRawDefaultValue(false);
    return false;
}
```

That sets the Fact default to `false` and tells QGC to not show the option under settings (retuning false).